gulp-src/lint-md.js

```diff
@@ -14,6 +14,7 @@ const markdownFiles = [
   '!layouts/**',
   '!node_modules/**',
   '!scripts/registry-scanner/node_modules/**',
+  '!scripts/generate-community-data/node_modules/**',
```

Comment:
This is fine as is. I'm curious if we could get away with `!**/node_modules/**`.

Reply:
We can, fixed it :-)